pybliometrics/scopus/serial_search.py

```diff
@@ -0,0 +1,303 @@
+from collections import namedtuple, OrderedDict
+
+from pybliometrics.scopus.superclasses import Search
+
+
+class SerialSearch(Search):
+    @property
+    def results(self):
+        """A list of namedtuples representing results of serial search. The
+        number of fields may vary from one search query to another depending
+        on the length of yearly data Note: can be empty.
+        """
+        out = []
+        search_results = self._json['serial-metadata-response'].get('entry')
+        if search_results:
```

Comment:
I think if you use `.get('entry', [])`, i.e return an empty list if there are no entries, you can remove one level of hierarchy, which greatly reduces complexity